__tests__/user/user.mutation.test.js

```diff
@@ -33,10 +33,18 @@ describe('mutations', () => {
 
         userId = res.data.registerUser._id
         
+        expect(res.data.registerUser).toHaveProperty(
+            'firstName', 'Petro'
+        );
+        expect(res.data.registerUser).toHaveProperty(
+            'lastName', 'Tatsenyak'
+        );
         expect(res.data.registerUser).toHaveProperty(
             'email', 'tacjka34@gmail.com'
         );
-
+        expect(res.data.registerUser).toHaveProperty('credentials');  
+        expect(res.data.registerUser).toHaveProperty('role');
+        expect(res.data.registerUser).toHaveProperty('registrationDate');
```

Comment:
Please, check that property `registrationDate` has equal to current date ignoring milliseconds and seconds

Reply:
fixed